playbooks/roles/lets-encrypt/templates/response.yaml.j2

```diff
@@ -0,0 +1,13 @@
+"acme-enter-email": "{{ streisand_admin_email }}"
+"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf": true
```

Comment:
I would prefer to not see this URL hardcoded. Let's Encrypt can change it at any time and new registrations will fail until this line is updated. Is it possible to prompt for it? I can think of some alternative workarounds if it will be tricky.

Reply:
@cpu I took the response file from acmetool's repo: https://github.com/hlandau/acme/blob/master/_doc/response-file.yaml .

I'm also a little uncomfortable on this, but I cannot think of a better way :-(

The agreement URL is subject to change, yes. But it's not like we can predict the future URL. So the only hope is that we watch out for announcements from Let's Encrypt and update the URL here when they change it.

Reply:
The URL changed 🤦‍♂️  https://github.com/StreisandEffect/streisand/pull/1053